client/scss/components/_icons.scss

```diff
@@ -69,9 +69,11 @@
   display: inline-block;
 }
 
-svg.icon-spinner {
-  // TODO: leave only class when iconfont styles are removed
+// Add spinner styles for SVG onto the use element,
+// so an icon can be turned into a spinner by changing the href only.
+use[href='#icon-spinner'] {
```

Comment:
tying to understand this approach? why not use classes?

Reply:
Spoke with Thibaud

* icon name added as the use element href
* icon name as a classname on the parent `svg`
* This approach allows us to avoid updating both when we want to trigger a spinner.

Maybe a room for improvement and discussion with UI team in the future or maybe we skip this change and raise a new issue for finding a good solution here.

header search field is where it is used

Reply:
Noted to the list of things to consider later.